Tests/UnitTests/Scripts/tests.js

```diff
@@ -616,16 +615,16 @@ describe("URLSearchParams", function () {
     });
 });
 
-*/
 describe("Console", function () {
-    it("should log a simple console log string without error", function () {
+  it("should log a simple console log string without error", function () {
         expect(() => console.log("I am a test string")).to.not.throw();
     });
-    it("should log sequence of strings", function () {
+  it("should log sequence of strings", function () {
         expect(() => console.log("I", "am", "a", "test", "string", 2, 2.345, { foo: 'bar' })).to.not.throw();
     });
     it("Should log string substitutions", function () {
-        expect(() => console.log("String sub: %s, float sub: %.2f, int sub: %2.d", "string", 3.1457, 3.1457)).to.not.throw();
+      expect(() => console.log("String sub: %s, float sub: %.2f, int sub: %2.d", "string", 3.1457, 3.1457)).to.not.throw();
+        //expect(() => console.log("float sub: %.2f, int sub: %2.d", 3.1457, 3.1457)).to.not.throw();
     });
     it("Should allow numbers to substitute strings", function () {
         expect(() => console.log("Print these numbers! %s %s", 1.2345, 1)).to.not.throw();
```

Comment:
nit: does it support emojis? not needed for this feqture but I'm curious.

Reply:
I'm not sure, is there a way to test for this?

Reply:
It does seem to work on the pipelines 😄 
<img width="560" alt="image" src="https://github.com/BabylonJS/JsRuntimeHost/assets/6002144/ac3b7f8c-032b-4bb2-a65b-5c50a6df2607">